stix2elevator/convert_stix.py

```diff
@@ -1618,6 +1667,74 @@ def negate_indicator(indicator):
     return hasattr(indicator, "negate") and indicator.negate
 
 
+def find_first_seen_sighting(sightings):
+    earlist_date = sightings[0].timestamp
+    for s in sightings:
+        if earlist_date:
+            if s.timestamp:
+                if s.timestamp <= earlist_date:
+                    earlist_date = s.timestamp
+        else:
+            earlist_date = s.timestamp
+    return earlist_date
+
+
+def find_last_seen_sighting(sightings):
+    last_date = sightings[0].timestamp
+    for s in sightings:
+        if last_date:
+            if s.timestamp:
+                if s.timestamp >= last_date:
+                    last_date = s.timestamp
+        else:
+            last_date = s.timestamp
+    return last_date
```

Comment:
Same for this except with `max()`:
```suggestion
    return max(sightings, key=attrgetter("timestamp")).timestamp
```